src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -343,23 +390,36 @@ private void create()
         throw new ContractValidateException(
             "feeLimit must be >= 0 and <= " + VMConfig.MAX_FEE_LIMIT);
       }
-
       AccountCapsule creator = this.deposit
           .getAccount(newSmartContract.getOriginAddress().toByteArray());
-      long energyLimit = getEnergyLimit(creator, feeLimit, callValue);
+
+      long energyLimit;
+      if (IS_NEW_VERSION) {  // TODO according to version
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbTWo_ILpqqSUlcalJp&open=AWbTWo_ILpqqSUlcalJp)

[ISSUE_KEY]: AWbTWo_ILpqqSUlcalJp